sdk/src/assertions/bmff_hash.rs

```diff
@@ -57,6 +74,63 @@ impl ExclusionsMap {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
Probably should move this into utils at some point, but ok here as long as it is only used here.